.github/workflows/build.yaml

```
@@ -44,10 +44,45 @@ jobs:
+    name: C# linting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Unity (from ProjectVersion.txt)
+        uses: buildalon/unity-setup@v1
+        with:
+          version-file: ProjectSettings/ProjectVersion.txt
+
+      - name: Activate Unity License (Personal)
+        if: ${{ secrets.UNITY_USERNAME != '' && secrets.UNITY_PASSWORD != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)